packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1137,7 +1137,7 @@ The first three parameters in the table below correspond to different types of l
 
 | Parameter   | Description                                                                                                                                    | Param Type   | DataType   | Required |
 | :---------- | :--------------------------------------------------------------------------------------------------------------------------------------------- | :----------- | :--------- | :------- |
-| q           | Finds a user that matches `firstName`, `lastName`, and `email` properties                                                                      | Query        | String     | FALSE    |
+| q           | Finds a user that matches `firstName`, `lastName`, and `email` properties. We recommend preferring search to this parameter    | Query        | String     | FALSE    |
 | filter      | [Filters](/docs/reference/core-okta-api/#filter) users with a supported expression for a subset of properties                  | Query        | String     | FALSE    |
 | search      | Searches for users with a supported [filtering](/docs/reference/core-okta-api/#filter) expression for most properties          | Query        | String     | FALSE    |
```

Comment:
should we indicate here that this is the recommended approach?

Reply:
Sounds good to me.